ci-scripts/teravm/fabfile.py

```diff
@@ -120,8 +120,34 @@ def upgrade_teravm(
 
     return hash
 
+def get_agw(setup):
+    version = None
+    """
+    Upgrade teravm agw to image with the given hash.
+    hash: a hash to identify what version from APT to use for upgrading.
+    If not hash provided or "latest" is passed, it will install latest
+    on the repository.
 
-def upgrade_teravm_agw(setup, hash, key_filename=DEFAULT_KEY_FILENAME):
+    key_filename: path to where the private key is for authorized-key based
+    ssh. The public key counterpart needs to in the authorized_keys file on
+    the remote host. If empty file name is passed, password-based ssh will
+    work instead. This can be used if the script is run manually.
+    """
+
+    fastprint("Upgrade teraVM AGW to %s")
```

Comment:
modify all the prints to describe what it is happening
Note that %s just indicates to print the content of a variable. If you dont have any variable to pass to the print, then you dont need the %s